todo/src/components/todo.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { toggleComplete } from "../actions";
+import { connect } from "react-redux";
+import "../App.css";
+
+class Todo extends Component {
+  toggleComplete(event) {
+    this.props.toggleComplete(this.props.index);
+  }
+
+  render() {
+    return (
+      <p className="todoList">
+        <input type="checkbox" onChange={this.toggleComplete.bind(this)} />
```

Comment:
```suggestion
        <input type="checkbox" onChange={this.toggleComplete} />
```
If we write toggleComplete as an arrow function we implicitly bind the value of this to Todo, and we don't have to explicitly bind it here